src/common/Pagination/index.js

```diff
@@ -0,0 +1,36 @@
+import { Button, PageChanger, PageCounter, Text } from "./styled";
```

Comment:
Zauwazylem, ze na pierwszej stronie gdy wyszukamy jakas fraze i mamy juz konkretne wyniki, to przejscie na ostatnia strone z paginacji nie powoduje bledu, czyli to rzeczywiscie musi byc kwestia api :P

Reply:
zgadza się, też to widziałam :)